modules/sb-uninstall-asb.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/uninstalling-ansible-service-broker.adoc
+
+[id="sb-uninstall-asb-{context}"]
+= Uninstalling the {asb-name}
+
+The following procedure uninstalls the {asb-name} and its Operator using the
+web console.
+
+WARNING: Do not uninstall the {asb-name} if there are any provisioned services from it in your cluster, otherwise you might encounter errors when trying to manage the services.
+
+.Prerequisites
+
+* The {asb-name} is installed.
+
+.Procedure
+
+This procedure assumes that you installed the {asb-name} into the
+`ansible-service-broker` project.
+
+. Uninstall the {asb-name}.
+.. Navigate to *Catalog* -> *Installed Operators* and select the
+*ansible-service-broker* project from the drop down.
+// TODO: Change to *OpenShift Ansible Service Broker Operator* by GA (4 of 5)
+.. Click *Automation Broker Operator*.
+.. Select the *Automation Broker* tab.
+.. Click *ansible-service-broker*.
+.. From the *Actions* drop down, select *Delete Automation Broker*.
+.. Click *Delete* from the confirmation pop up.
```

Comment:
s/pop up/page (or window maybe?) IBM style states that we should avoid use of "pop up".